.github/ISSUE_TEMPLATE/job-story.md

```
@@ -0,0 +1,206 @@
+Requirements for the different teams which participate at the specific story\
+Known Limitations
+-----------------
+
+Brainstorm things that could go wrong with your team and partner teams. For each risk, plan appropriate mitigations.\
+**Rough Scoping & Timeline**
+
+At a high level, what's included in V1 vs. later versions? How big of a project is this? What's the roll out / testing plan? Consider the major pieces of functionality, Mobile, Platform, Internationalization, Entry Points, User Onboarding, Premium.
+
+This is part of  the feasibility and Stakeholders of all departments have to attend and be accountable for their department decisions. 
+
+**Success Criteria**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)